docs/introduction/toolbar.rst

```diff
@@ -60,49 +49,34 @@ Create a new ``cms_toolbars.py`` file in your Polls/CMS Integration application:
                 url=admin_reverse('polls_poll_add'),
             )
 
+
+    toolbar_pool.register(PollToolbar)  # register the toolbar
+
+
 .. note:: Don't forget to restart the runserver to have your new toolbar item recognised.
 
 
 What this all means
 ===================
 
-Right now you don't need to know exactly what this does, but once you've got it working here's
-a quick run-down what the code above does:
 
-* defines a ``CMSToolbar`` sub-class
-* registers the toolbar class with ``@toolbar_pool.register``
-* defines a ``populate()`` method that adds an item to the menu
-
-The ``populate()`` method:
+``CMSToolbar`` sub-classes must implement a ``populate()`` method. The ``populate()`` method will only be called if the
+current user is a staff user. The ``populate()`` method is what actually manages the toolbar - it:
```

Comment:
maybe would be good to clarify "method is what actually manages the toolbar - it:" to say "method is what actually manages the toolbar - in our case, it will:"

Reading through it, was a bit confusing to see the poll app specific stuff in `populate()`.